tools/protein_properties/test-data/protein_properties_result1.bed

```diff
@@ -0,0 +1,3 @@
+ID	Sequence	MW	IP	gravy	length	instability	monoisotpoic
+sp|Q9LHB9|PER32_ARATH	MNFSYSSLSTWTTLMTLGCLLLHSSISSAQLTPTFYDNTCPSVFTIVRDTIVNELRSDPRIAASILRLHFHDCFVNGCDASILLDNTTSFRTEKDAAPNANSARGFPVIDRMKAAVETACPRTVSCADILTIAAQQAVNLAGGPSWRVPLGRRDSLQAFFALANTNLPAPFFTLPQLKASFQNVGLDRPSDLVALSGGHTFGKNQCQFIMDRLYNFSNTGLPDPTLNTTYLQTLRGQCPRNGNQTVLVDFDLRTPTVFDNKYYVNLKELKGLIQTDQELFSSPNATDTIPLVREYADGTQKFFNAFVEAMNRMGNITPLTGTQGQIRQNCRVVNSNSLLHDVVEIVDFVSSM	38846.6361	6.20635986328	-0.0943181818182	352	31.0301420455	0.0880681818182
```

Comment:
is it useful to have the sequence in this? Makes the output harder to read for long protiens...